pom.xml

```diff
@@ -437,6 +437,18 @@
   </properties>
 
   <dependencies>
+
+    <dependency>
+      <groupId>com.nerdynick</groupId>
+      <artifactId>commons-configuration2-vault</artifactId>
```

Comment:
what is the purpose of this dependencies? as vault integration is really a core future of this, I tend to thing it's code base should land directly in the project. What do you think?

Reply:
This is a module I wrote for another project that implements vault against the apache config2 interfaces. So it makes no since to rewrite or direct import it.